features/autoconsent.json

```diff
@@ -294,6 +294,10 @@
         {
             "domain": "97x.com",
             "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2304"
+        },
+        {
+            "domain": "97x.com",
```

Comment:
@icodebyamanda this should be speedweek.com, no?

Reply:
@dharb apologies, just made the change and thanks for checking it!